content/en/docs/advanced-tasks/adding-admission-and-conversion-webhooks.md

```diff
@@ -0,0 +1,138 @@
+
+This example will focus on installing an operator that the OLM team built for it's e2e test portfolio, the [webhook-operator](https://github.com/awgreene/webhook-operator/tree/olm). The webhook-operator implements a validating, mutating, and conversion webhook for the [WebhookTest CRD](https://github.com/awgreene/webhook-operator/blob/olm/config/crd/bases/webhook.operators.coreos.io_webhooktests.yaml).
+
+#### Defining Conversion Webhooks in the ClusterServiceVersion
+
+The full webhook operator CSV can be seen [here](https://github.com/awgreene/webhook-operator/blob/07375018e32cbebedee55aa1b131469c247dd275/bundle/manifests/webhook-operator.clusterserviceversion.yaml), but we will highlight the important parts of the CSV below:
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: webhook-operator.v0.0.1
+spec:
+  customresourcedefinitions:
+    owned:
+    - kind: WebhookTest
+      name: webhooktests.webhook.operators.coreos.io # The CRDs target by the conversion webhook must exist here
+      version: v1
+  install:
+    spec:
+      deployments:
+      - name: webhook-operator-webhook
+        ...
+        ...
+        ...
+    strategy: deployment
+  installModes:
+  - supported: false
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  webhookdefinitions:
+  - type: ValidatingAdmissionWebhook
+    admissionReviewVersions:
+    - v1beta1
+    - v1
+    containerPort: 443
+    targetPort: 4343
+    deploymentName: webhook-operator-webhook
+    failurePolicy: Fail
+    generateName: vwebhooktest.kb.io
+    rules:
+    - apiGroups:
+      - webhook.operators.coreos.io
+      apiVersions:
+      - v1
+      operations:
+      - CREATE
+      - UPDATE
+      resources:
+      - webhooktests
+    sideEffects: None
+    webhookPath: /validate-webhook-operators-coreos-io-v1-webhooktest
+  - type: MutatingAdmissionWebhook
+    admissionReviewVersions:
+    - v1beta1
+    - v1
+    containerPort: 443
+    targetPort: 4343
+    deploymentName: webhook-operator-webhook
+    failurePolicy: Fail
+    generateName: mwebhooktest.kb.io
+    rules:
+    - apiGroups:
+      - webhook.operators.coreos.io
+      apiVersions:
+      - v1
+      operations:
+      - CREATE
+      - UPDATE
+      resources:
+      - webhooktests
+    sideEffects: None
+    webhookPath: /mutate-webhook-operators-coreos-io-v1-webhooktest
+  - type: ConversionWebhook
+    admissionReviewVersions:
+    - v1beta1
+    - v1
+    containerPort: 443
+    targetPort: 4343
+    deploymentName: webhook-operator-webhook
+    generateName: cwebhooktest.kb.io
+    sideEffects: None
+    webhookPath: /convert
+    conversionCRDs:
+    - webhooktests.webhook.operators.coreos.io # Make sure that each CRD defined in this list has the spec.PreserveUnknownFields property set to `false` or nil.
```

Comment:
Nit: Can we trim the comment to be a bit short or line-wrap it somehow? I think it is too long to fit in the UI it seems.